hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/delegation_tokens.md

```diff
@@ -108,7 +108,7 @@ password-protected data opaque to clients; they contain the secrets needed
 to access the relevant S3 buckets and associated services.
 
 They are obtained by requesting a delegation token from the S3A filesystem client.
-Issued token mey be included in job submissions, passed to running applications,
+Issued tokens may be included in job submissions, passed to running applications,
 etc. This token is specific to an individual bucket; all buckets which a client
 wishes to work with must have a separate delegation token issued.
```

Comment:
I find this confusing in the context of delegated role tokens, where the token is _literally_ specific to an individual bucket.

Can we make this clearer while we are editing this paragraph?

```suggestion
Issued tokens may be included in job submissions, passed to running applications,
etc.
Delegation tokens are specific to an S3A filesystem.
Each filesystem a client wishes to work with must have a separate delegation token issued,
regardless of the delegation token variant.
```

Is it correct to describe them specific to a filesystem rather than the bucket?

Reply:
Discussed offline - will leave it as is for now as the suggestion wouldn't be making it clearer.